src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -256,11 +252,22 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
     runTransaction(action)
   }
 
+  private def insertExecutionInfos(insertResult: this.dataAccess.executionsAutoInc.SingleInsertResult,
```

Comment:
I'd prefer `insertEmptyExecutionInfos` - I was staring at this for ages until I realised it's just creating empty items so that all subsequent calls don't need to worry about INSERT vs UPDATE